x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -45,10 +45,17 @@ Please, before opening a PR with the new test, please make sure that the test fa
 
 ## Running the tests
 
-There are currently four ways to run the tests, comprised of two execution modes and two target environments, which will be detailed below.
+### Run them locally
+Run the tests locally with the following command:
+
+| Command | Description |
+| -- | -- |
+| `yarn cypress:open` | Starts a local ES and Kibana and runs the Cypress binary. All e2e specs are available to run via the UI |
 
 ### Execution modes
 
+There are currently four ways to run the tests, comprised of two execution modes and two target environments, which will be detailed below.
```

Comment:
where below exactly? I was expecting perhaps some CLI args to add?

Reply:
Sorry I don't have any more details. I moved this line from earlier in the document. I think this readme needs a thorough review from a few stakeholders.